packages/sdk/src/ZSshUtils.ts

```diff
@@ -34,5 +48,94 @@ export class ZSshUtils {
         return typeof data !== "string" ? Buffer.from(data).toString("base64") : data;
     }
 
-    public static deployServer(): void {}
+    public static async installServer(session: SshSession, serverPath: string, localDir: string): Promise<void> {
+        const client = new Client();
+        client.connect(ZSshUtils.buildSshConfig(session));
+        return new Promise((resolve, reject) => {
+            client.on("ready", () => {
+                client.sftp((err, sftp) => {
+                    if (err) {
+                        reject(err);
+                    } else {
+                        ZSshUtils.uploadDir(sftp, localDir, serverPath.replace(/^~/, "."))
+                            .then(resolve, reject)
+                            .finally(() => client.end());
+                    }
+                });
+            });
+        });
+    }
+
+    public static async uninstallServer(session: SshSession, serverPath: string, localDir: string): Promise<void> {
```

Comment:
The install and uninstall methods look almost identical, should we combine then and just pass the zsshUtils function to be called? 😋

Reply:
Resolved in https://github.com/zowe/zowe-native-proto/pull/34/commits/298f1f92f1df600fae04fadc60fb97fe2894914a - kept separate methods but reduced duplication